sorl_watermarker/engines/base.py

```diff
@@ -66,7 +67,15 @@ def watermark(self, image, options):
             raise AttributeError('Trying to apply a watermark, '
                                  'however no THUMBNAIL_WATERMARK defined, '
                                  'and watermark not set on tag')
+        # Note - I do not think find can be used for this purpose - it is used for collecting static files to the static folder
+        # Use os.path.join with the static root instead, and if not found, raise the error.
```

Comment:
If you can confirm that removing #72 doesn't violate the workflow on Django1.6-1.9 under python2.7 and python3.4 interpreters, you are welcomed to fully replace the "find" method call with os.path.join :)